odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW

```diff
@@ -949,7 +987,52 @@ PROCEDURE READCONFIGFILESETTINGS
  FOR TMPLOOP=0 TO MAXSLTYPES
   DO
    VALIDSHARETYPES(TMPLOOP)=LELIST(TMPLOOP)
-  END
+
+[* If share type is eligible, then loop through the share types
+** defined in the OTA parameter setting and if a matching share type
+** has been defined, leave the OTA amount at that value, else if the
+** OTA default amount has been defined, set the OTA amount to that
+** value, else, leave it at the initialization value of $0.00.
+*]
+   IF VALIDSHARETYPES(TMPLOOP)=TRUE THEN
+    DO
+     SHARETYPEFOUND=FALSE
+     OTAAMOUNT=$0.00
+     FOR OTALOOP=1 TO PARAMOTACOUNT
+      DO
+       IF PARAMOTASHARETYPE(OTALOOP)=TMPLOOP THEN
+        DO
+         SHARETYPEFOUND=TRUE
+         OTAAMOUNT=PARAMOTAAMOUNT(OTALOOP)
+        END
+      END [FOR OTALOOP=1 TO PARAMOTACOUNT]
+
+     BNODEBUGMSG=FORMAT("Evaluating OTA parm list for share type 9999",TMPLOOP)+
+                 "  SHARE TYPE FOUND="+BNOTRUEFALSECHR(SHARETYPEFOUND)
+
```

Comment:
It appears you are missing a call to the BNODEBUGMSGADD procedure. I did test, and this "Evaluating OTA..." line is never sent to the output. Also, in that same line, I think you have misspelled param (parm)

Reply:
@jucarson Corrections made - thanks!